styles/components/Icon.module.scss

```diff
@@ -0,0 +1,11 @@
+.blue{
```

Comment:
These styles should go in your components stylesheet. See constants.scss for app colors.

Reply:
Removed this file in the latest revision.